en/docs/reference/config-catalog.md

```diff
@@ -4389,3 +4389,59 @@ persistent_notifier.password = "root"</code></pre>
         </div>
     </section>
 </div>
+<<<<<<< HEAD
+=======
+
+## Tenant Management
+
+<div class="mb-config-catalog">
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+                <input name="16" type="checkbox" id="_tab_16">
+                <label class="tab-selector" for="_tab_16"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">

```

Comment:
@Chamindu36 , Nothing appears when the "view sample" button is clicked.

<img width="785" alt="Screenshot 2020-10-14 at 15 54 00" src="https://user-images.githubusercontent.com/5195851/95976729-932a2380-0e35-11eb-9e6e-0474398037d8.png">

Reply:
Resolved